RainwayApp/flutter_gamepad
# Android support

Adds Android support to the gamepad plugin.

@apecoraro @nir-ziv To test:
* Build the example app in this package on your Android device.
* Connect a Bluetooth gamepad to your device.
* Right before the first registered button press, you should see a "connected gamepad" event in the example app.
* Test all the buttons: A, B, X, Y, shoulders, stick buttons, Menu, and the directional pad. They should all produce log entries in the app. The B button should not close the app.
  * ("Select/Back" will always perform a UI action on Android, I'm afraid.)
* Test the joysticks and trigger buttons. Moving them should also produce log entries in the app.
* Test the "Call gamepads()" button in the example app: your controller's name should show up.

Basically, you should see something like this:

![image](https://user-images.githubusercontent.com/16232127/69180134-156a7480-0b0d-11ea-99ac-793876d18938.png)
